SmartDeviceLink/SDLError.h

```diff
@@ -24,6 +24,7 @@ extern SDLErrorDomain *const SDLErrorDomainSoftButtonManager;
 extern SDLErrorDomain *const SDLErrorDomainMenuManager;
 extern SDLErrorDomain *const SDLErrorDomainChoiceSetManager;
 extern SDLErrorDomain *const SDLErrorDomainTransport;
+extern SDLErrorDomain *const SDLErrorDomainStore;
```

Comment:
Please update to `SDLErrorDomainRPCStore`

Reply:
fixed in 064456a